apps/hashdotdev/src/components/anchor-tag.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { NAV_HEIGHT } from "./navbar";
+
+export const AnchorTag: FunctionComponent<{ id: string }> = ({ id }) => (
```

Comment:
Personally I'd name this `AnchorFragmentTag` or something more specific (maybe even throw 'hidden' in there), since any `<a>` is an anchor tag

Reply:
Fair point, have renamed to `HiddenAnchorFragmentTag` in https://github.com/hashintel/hash/pull/2747/commits/23b5abb89eef42cf16cb6e1d11bcddbcebee9a39